packages/codemods/README.md

```diff
@@ -1,6 +1,20 @@
-# ts-morphin-migration
+# @fluentui/codemods
 
-ts-morphin-migration is a package created and managed with the `create-just` utility.
+This is a utility package to assist with the upgrading of libraries and apps that rely on fluentui.
+
+## How it works
+
+If you have a typescript application or library that relies on fluentui then you can run `npx @fluentui/codemods` which will immediately begin an upgrade of your codebase so that you don't have to! It works by finding all the tsconfig files and then using those to find the relevant files to upgrade before running the updates on each of them!
```

Comment:
I still think it would be helpful to specify the particular package names it's capable of upgrading (in particular that OUFR is supported too)

Reply:
Currently nothing is explicitly supported, we're still figuring out how all of that will work.